includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -527,8 +531,9 @@ private function get_courses( $args ) {
 	/**
 	 * Return array of lessons
 	 *
-	 * @since  1.7.0
+	 * @param object $args Arguments for query.
 	 * @return array lessons
+	 * @since  1.7.0
 	 */
```

Comment:
It would be nice to try and follow a similar practice for docblocks. What I'm trying to follow is the [WP DocBlock Formatting standard](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods) (with the exception of the newline before the return 😄 ).

```php
/**
 * Description.
 *
 * @since 1.7.0
 * @deprecated 3.1.0 In favor of this other method.
 *
 * @see https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods
 *
 * @param array $first Param description.
 * @param string $second Optional. Param description. Default.
 *
 * @return array Return description.
 */
```

Reply:
Thanks, will try to follow this from now on, updated this instance here https://github.com/Automattic/sensei/pull/4883/commits/924df553acadf51176832236234004ca471760bf